### PR TITLE
Sigil generator on Point Detail Screen

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
Adds customisable sigil generator to point detail screen.

Drafting as this is based off the Cohort Screens, so blocked pending that merge.

### CODE REVIEW

File: src/views/SigilGenerator.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import { useLocalRouter } from 'lib/LocalRouter';
+import useSigilDownloader from 'lib/useSigilDownloader';
+
+import { composeValidator, buildNumberValidator } from 'form/validators';
+import BridgeForm from 'form/BridgeForm';
+import FormError from 'form/FormError';
+import SubmitButton from 'form/SubmitButton';
+import { NumberInput, ColorInput } from 'form/Inputs';
+
+const BG_COLORS = [
```
Comment (jyng) [inline src/views/SigilGenerator.js]: Could these be imported from indigo-tokens?

File: src/form/Inputs.js [hunk 2]
```diff
@@ -196,3 +197,53 @@ export function EmailInput({ ...rest }) {
     />
   );
 }
+
+export const ColorInput = ({
```
Comment (jyng) [inline src/form/Inputs.js]: Hm, why are we not using the indigo-react/inputs here?

File: src/form/SubmitButton.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ export default function SubmitButton({
   handleSubmit,
   ...rest
 }) {
+  const form = useFormState();
```
Comment (jyng) [inline src/form/SubmitButton.js]: Could we try using the indigo-react/buttons here?

File: src/form/Inputs.js [hunk 4]
```diff
@@ -197,53 +196,3 @@ export function EmailInput({ ...rest }) {
     />
   );
 }
-
-export const ColorInput = ({
```
[the remark this answers is not in the document]
Comment (liam-fitzgerald) [reply src/form/Inputs.js]: So move this back into form/Inputs.js ? Indigo-react doesn't currently contain a color picker

[the remark this answers is not in the document]
Comment (jyng) [reply src/form/Inputs.js]: Just off the top of my head, I would move color picker src/components (which im assuming are local components) and then inside the colorPicker.js I would use atomic components from indigo-react.

**Pull request merged**